modules/service/src/main/scala/lucuma/odb/service/F2LongSlitService.scala

```diff
@@ -98,7 +110,7 @@ object F2LongSlitService {
           ls.c_decker,
           ls.c_readout_mode
         FROM
-          #t_flamingos_2_long_slit ls
+          t_flamingos_2_long_slit ls
```

Comment:
I'm not sure why the gmos queries have a `#`. Why's that?

Reply:
Since the two queries (north and south) are the same except for the table name, the query uses literal string interpolation to identify the table and not repeat the rest.  Here's an example from the skunk doc:

```
val table = "my_table"
val frag  = sql"SELECT foo, bar FROM #$table where foo = $int4"

frag.sql
// res4: String = "SELECT foo, bar FROM my_table where foo = $1"
```